packages/amplication-client/src/Workspaces/WorkspaceHeader/GitHubBanner.tsx

```diff
@@ -20,9 +20,8 @@ export default function GitHubBanner() {
   const isBannerShowing = JSON.parse(showBanner);
 
   const handleBannerClick = (
-    eventName:
-      | AnalyticsEventNames.StarUsBannerCTAClick
-      | AnalyticsEventNames.StarUsBannerClose
+    eventName: // | AnalyticsEventNames.StarUsBannerCTAClick
+    AnalyticsEventNames.StarUsBannerClose
```

Comment:
keep both events, but only close the banner on the specific one. 

you can use two different callbacks, something like


const handleStarUsClick = useCallback(()=>{
trackEvent({
      eventName: AnalyticsEventNames.StarUsBannerCTAClick
    });
})

const handleCloseStarUsClick = useCallback(()=>{
setShowBanner("false");
trackEvent({
      eventName: AnalyticsEventNames.StarUsBannerClose
    });
})